### ISSUE TITLE
French Translation

### ISSUE THREAD

**Issue by mdevlamynck** [GabuEx/my-little-investigations#13]
Hi,

I have a friend willing to do the french translation of the game. Where can I find the text of the dialogs ?

**Comment by mdevlamynck** [comment]
I've translated the game localized content and I'll move on to translate the first case content.

You can find the [common_fr-FR.dat here](http://www.art-software.fr/files/MLI/common_fr-FR.dat). Note that I had to use an alternative font as JennaSue.ttf doesn't support accentuated characters. After a little research I ended up using ShadowOverLight wich is under a free license (included in the .dat archive). There may exist a closer looking font though.

**Comment by GabuEx** [comment]
Sweet!  Awesome to see another translation coming. :)  The font you currently have selected looks OK to me, though you may want to reword some things in the options menu, since it looks pretty wonky at the moment.

Hmm, come to think of it, I probably want to make it so that translations can translate the names of other translations, given that I hadn't thought of the fact that having every translation appearing at the same time such that they all need to be able to display in a way that's displayable with the currently selected font.  I'll look into doing that.

**Comment by mdevlamynck** [comment]
Good call, just keep a default translation in case a new language appears and the others aren't up to date to handle it.

**Comment by GabuEx** [comment]
Yeah, I'll think about what the best way is to handle that, that's tricky.  Though it helps that I can synchronize them such that they can be uploaded together once they're ready.

**Comment by mdevlamynck** [comment]
Can't you have a fallback to english when a translation is missing from some other language ?

**Comment by GabuEx** [comment]
That was more or less what I was thinking of, yeah.
